modules/quotas-selecting-projects.adoc

```diff
@@ -66,6 +67,7 @@ pods and 20 secrets.
 
 . Similarly, to select projects based on labels, run this command:
 +
+[source,terminal]
 ----
 $  oc create clusterresourcequota for-name \ <1>
```

Comment:
The rendered version of this loses color-coding after the first line. See [this example](https://github.com/openshift/openshift-docs/blame/master/modules/customize-certificates-api-add-named.adoc#L36-L42) for how this can be solved:

Reply:
Thank you for highlighting this and for the pointer. Resolved.